views/emails/postMentioned.blade.php

```diff
@@ -1,9 +1,8 @@
-Hey {!! $user->display_name !!}!
-
-{!! $blueprint->reply->user->display_name !!} replied to your post (#{!! $blueprint->post->number !!}) in {!! $blueprint->post->discussion->title !!}.
-
-{!! app()->url() !!}/d/{!! $blueprint->reply->discussion_id !!}/{!! $blueprint->reply->number !!}
-
----
-
-{!! $blueprint->reply->content !!}
+{!! $translator->trans('flarum-mentions.email.post_mentioned.body', [
+'{recipient_username}' => $user->display_name,
```

Comment:
Shouldn't this key be named `displayname` or `display_name` for consistency?

Reply:
recipient_display_name then?

Reply:
Yeah, that would be my suggestion. Not sure what other think about it.

This would also need to be changed in the other PRs.

Reply:
recipient_username is also used in https://github.com/flarum/subscriptions/pull/30

I think recipient_display_name is best, but please modify that in both PR's.